tests/Resources/ti.ui.imageview.test.js

```diff
@@ -0,0 +1,367 @@
+			width: Ti.UI.FILL, height: Ti.UI.FILL
+		});
+		imageView.addEventListener('start', function () {
+			try {
+				should(imageView.animating).be.true;
+			} catch (err) {
+				error = err;
+			}
+
+			finish(error);
+		});
+		imageView.addEventListener('load', function () {
+			try {
+				should(imageView.animating).be.false;
+				imageView.start();
+			} catch (err) {
+				error = err;
+			}
+		});
+		win.addEventListener('open', function () {
+			var fromFile = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'Logo.png');
+			imageView.images = [ fromFile, fromFile, fromFile ];
+		});
+
+		win.add(imageView);
+		win.open();
+	});
+
+	// iOS Can't read the image from the Blob. Fix how we grab the blob on iOS?
+	// Android crashes for me locally. We should load blobs from multiple files, not the same one multiple times
+	it.androidAndIosBroken('images (Blob)', function (finish) {
+		var imageView,
+			error;
+		this.timeout(10000);
+
+		win = Ti.UI.createWindow();
+		imageView = Ti.UI.createImageView({
+			width: Ti.UI.FILL, height: Ti.UI.FILL
+		});
+
+		imageView.addEventListener('start', function () {
+			try {
+				should(imageView.animating).be.true;
+			} catch (err) {
+				error = err;
+			}
+
+			finish(error);
+		});
+		imageView.addEventListener('load', function () {
+			try {
+				should(imageView.animating).be.false;
+			} catch (err) {
+				error = err;
+			}
+			imageView.start();
+		});
+		win.addEventListener('open', function () {
+			var fromFile = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'Logo.png');
+			imageView.images = [ fromFile.read(), fromFile.read(), fromFile.read() ];
+		});
+
+		win.add(imageView);
+		win.open();
+	});
+
+	// TIMOB-18684
+	// FIXME Get working on iOS. Times out. never fires postlayout?
+	// FIXME Times out on Android build agent. likely postlayout never fires
+	it.androidAndIosBroken('layoutWithSIZE_and_fixed', function (finish) {
+		var view,
+			innerView;
+		this.slow(1000);
+		this.timeout(10000);
+
+		win = Ti.UI.createWindow();
+
+		view = Ti.UI.createView({
+			backgroundColor: 'green',
+			width: 100,
+			height: Ti.UI.SIZE
+		});
+		innerView = Ti.UI.createImageView({
+			image: 'http://api.randomuser.me/portraits/women/0.jpg',
+			width: 100,
+			height: Ti.UI.SIZE,
+			top: 0,
+			left: 0
+		});
+		view.add(innerView);
+		view.addEventListener('postlayout', function () {
+			try {
+				should(innerView.size.height).eql(100);
+				should(view.size.height).eql(innerView.size.height);
+				should(view.size.width).eql(innerView.size.width);
+
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+		win.add(view);
+		win.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('image from density specific folder on Android', function (finish) {
+		var window = Ti.UI.createWindow({backgroundColor: 'gray'});
+		var imageView = Ti.UI.createImageView({width: '50%', image: '/images/git.png'});
+		imageView.addEventListener('load', function() {
```

Comment:
I think this will never fire because the image is local and has loaded in the create method. You may need to hook the listener first and then assign the image property after?